.gitignore

```diff
@@ -12,6 +12,7 @@
 .editorconfig
 .gopath/
 .go-pkg-cache/
+.vscode/
```

Comment:
@AkihiroSuda thank you for this global .gitignore solution and that's much cooler.